sdk/notificationhubs/notification-hubs/rollup.test.config.cjs

```diff
@@ -25,7 +24,6 @@ function makeBrowserTestConfigPatch() {
     typescript({
       exclude: ["test/**/*.spec.ts"],
     }),
-    sourcemaps(),
```

Comment:
Does this damage the source map output at all? For folks using the ESM imports it won't matter, but what about CJS? I thought we relied on this plugin to unify our source maps. Is there no alternative in the rollup 3 universe?

Reply:
This is for testing purposes only